app/controllers/distance_service_app.rb

```diff
@@ -0,0 +1,26 @@
+require './app/services/map_quest_service'
+require './app/poros/distance_poro'
+require './app/serializers/distance_serializer'
+require './app/facades/distance_facade'
+require 'sinatra'
+require 'json'
+require 'fast_jsonapi'
+require 'sinatra/json'
+
+class DistanceServiceApp < Sinatra::Base
+  #get '/' do
+  #   "Hello, World!"
+  #end
```

Comment:
Did you mean to remove this commented out code? or is this left here intentionally?

Reply:
I did. Katy added a root which would specify that path, it was also just to confirm I was at 100% test coverage with the methods I created.